src/main/webapp/i18n/de/global.json

```diff
@@ -26,6 +26,7 @@
                 "modelingExercise": "Modeling Exercise",
                 "quizExercise": "Quiz Exercise",
                 "lecture": "Vorlesungen",
+                "learningGoal": "Lernziele",
```

Comment:
Even though the JSON ist only a goal (no plural), the translation is plural

Reply:
It's the same for lectures and exercises, that's why I kept it consistent.